public/app/features/dashboard/state/PanelModel.test.ts

```diff
@@ -165,7 +165,7 @@ describe('PanelModel', () => {
       it('should call react onPanelTypeChanged', () => {
         expect(onPanelTypeChanged.mock.calls.length).toBe(1);
         expect(onPanelTypeChanged.mock.calls[0][1]).toBe('table');
-        expect(onPanelTypeChanged.mock.calls[0][2].fieldOptions).toBeDefined();
+        expect(onPanelTypeChanged.mock.calls[0][2].angular).toBeDefined();
```

Comment:
This changes the test to make sure an `angular` object is passed to it.

I don't think the previous expectation is correct -- the `old` properties when changing from an angular panel would never have `fieldOptons`, but the setup (changed in the PR) applied defaults *before* asking the onPanelTypeChanged callback what it should do